scripts/plot_pole_zero.py

```
@@ -1,24 +1,53 @@
 #!/usr/bin/env python3
-"""Plot pole-zero diagrams from pole_positions.csv.
+"""Pole-zero displacement visualization.
 
-Unit circle with reference poles (filled) and displaced poles (open markers).
-One subplot per filter family. Displacement vectors drawn between positions.
+Reads `pole_positions.csv` and produces two figures:
+
+- `pole_zero.{png,pdf}` — unit-circle diagram with reference (double)
+  poles as large black × markers, quantized poles as smaller colored
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)